src/frontendHighlighterApp/sass/app.scss

```
@@ -823,6 +823,215 @@ body {
+			white-space: nowrap !important;
+			flex-shrink: 0 !important;
+
+			&--landmark {
+				background-color: #e8f0f8 !important;
+				color: variables.$color-blue-dark !important;
+			}
+		}
+
+		.edac-structure-item-text {
+			color: #1e1e1e !important;
+			word-break: break-word !important;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)